### PR TITLE
Fix problem in insertClassReferenceForModule function (issue #28)

### PR DESCRIPTION
**Pull request by mishasmaga**

### CODE REVIEW

**Comment by mishasmaga** [comment]
@angelwen08 , can you review a PR?

**Comment by qinezh** [comment]
@mishasmaga  can you please resolve the conflicts?

File: gulpfile.js [hunk 1]
```diff
@@ -12,6 +12,6 @@ gulp.task("default", function () {
             formatter: "verbose"
         }))
         .pipe(tslint.report())
-        .pipe(tsProject())
+        .pipe(tsProject())
```
Comment (qinezh) [inline gulpfile.js]: remove whitespaces?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "type2docfx",
-  "version": "0.9.114",
+  "version": "0.9.115",
```
Comment (qinezh) [inline package.json]: Please keep the version unchanged.

**Pull request closed without merging**